### ISSUE TITLE
Gamepad support

### ISSUE THREAD

**Issue by d-513** [PolyMC/PolyMC#209]
The main UI should be navigable with a gamepad. This would also bring better support for Steam Deck devices.

Possibly also offer to install LambdaControls when a controller is present to enable controller support in Minecraft

https://doc.qt.io/qt-5/qtgamepad-index.html

**Comment by LennyMcLennington** [comment]
Making it actually pleasant to navigate with a gamepad may require a redesign of the whole UI. Making it nagivable with a gamepad could probably be done without fully redesigning the UI.

**Comment by d-513** [comment]
> Making it actually pleasant to navigate with a gamepad may require a redesign of the whole UI. Making it nagivable with a gamepad could probably be done without fully redesigning the UI.

Ideally since it's QT a framework would handle it for us with minimal effort, I don't think that's what QTGamepad does though.

Yeah UI redesign would be best but lot of effort

**Comment by dada513** [comment]
@LennyMcLennington Lenny I got banned from discord can you unban me I was testing anti spam

**Comment by ZekeZDev** [comment]
could we make it work with the kde tablet ui? I am pretty sure thats part of what devices like the steam deck use. A greater overhaul would probably be needed in the future though

**Comment by Merith-TK** [comment]
Nope, the steam deck only uses KDE in desktop mode, and that is the full support of KDE
